sdk/servicebus/service-bus/tsconfig.json

```diff
@@ -30,7 +29,8 @@
     /* Other options */
     "newLine": "LF" /*	Use the specified end of line sequence to be used when emitting files: "crlf" (windows) or "lf" (unix).”*/,
     "allowJs": false /* Don't allow JavaScript files to be compiled.*/,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "downlevelIteration": true
```

Comment:
Can you explain the need for the new config here?

Reply:
There was a compilation error that specifically required for this to be added.

Reply:
Can you share what that error was? Since this is not part of the template, I am curious to know what required this and if we should add it to the template

Reply:
The error isn't showing now, removed this